README.md

```diff
@@ -149,6 +180,8 @@ pytest tests
 
 again from the root of the repository.
 
+N.B. Currently no tests exist.
+
```

Comment:
```suggestion
N.B. Currently no tests exist.

```
Perhaps remove this and open an issue to add tests, just so we don't forget to update this if / when tests are added